using-kubecost/navigating-the-kubecost-ui/savings/container-request-right-sizing-recommendations.md

```diff
@@ -1,46 +1,65 @@
 
-Select _Savings_ in the left navigation, then select _Right-size your container requests_. The Request right-sizing recommendations page opens.
+![Detailed view of a container where Kubecost has a recommendation](/images/crss-workload-savings.png)
+
+## Prerequisites
+
+While there are no restrictions to provide/view container request right-sizing recommendations, in order for Kubecost to apply those recommendations the [Cluster Controller](/install-and-configure/advanced-configuration/controller/cluster-controller.md) component is required. You must deploy Cluster Controller to the cluster(s) where Kubecost will apply any recommendations.
+
+## Configuring request right-sizing recommendations
 
 Select _Customize_ to modify the right-sizing settings. Your customization settings will tell Kubecost how to calculate its recommendations, so make sure it properly represents your environment and activity:
 
-* Window: Duration of deployment activity Kubecost will observe
-* Profile: Select from _Development_, _Production_, or High Availability\*, which come with preconfigured values for CPu/RAM target utilization fields. Selecting _Custom_ will allow you to manually configure these fields.
-* CPU/RAM recommendation algorithm: Always configured to _Max_.
-* CPU/RAM target utilization: Refers to the percentage of used resources over total resources available.
-* Add Filters: Optional configuration to limit the deployments which will have right-sizing recommendations applied. This will provide greater flexibility in optimizing your environment. Ensure you select the plus icon next to the filter value text box to add the filter. Multiple filters can be added.
+* Window: Duration of deployment activity Kubecost will observe.
+* Profile: Select from _Development_, _Production_, or High Availability, which come with predefined values for CPU/RAM target utilization fields. Selecting _Custom_ will allow you to manually configure these fields.
+* CPU/RAM recommendation algorithm: Set to _Max_ when any non-custom profile is in use. Additional algorithms include percentile of average and percentile of max.
+* CPU/RAM target utilization: Used along with the recommendation algorithm to determine the final value used for the new requests. For example, when the recommendation algorithm is configured for _Max_, a target utilization of 0.5 will ensure Kubecost recommends request values which are no higher than 50% of the observed maximum usage across the observed time window.
+* Add Filters: Optional configurations to limit the returned request right-sizing results. Multiple filters are supported.
 
 When finished, select _Save_.
 
-Your configured recommendations can also be downloaded as a CSV file by selecting the three dots button > _Download CSV_.
+{% hint style="info" %}
+Regardless of how the recommendations are configured or how low the observed utilization is, Kubecost has a minimum recommendation threshold of 10m for CPU and 20Mi for RAM.
```

Comment:
Thank you for adding this! I get this question a lot.

Reply:
Here is also the reasoning as to _why_ those are the defaults. Up to you whether we want to include that here! https://github.com/kubecost/kubecost-cost-model/blob/7142c6e446296965c9bc2af142b695ceb3d8f4a0/pkg/savings/requestsizing/v2/handlers.go#L32-L36

Reply:
Good to know but maybe a bit too in the weeds for general purpose docs.